mm2src/kdf_walletconnect/src/chain.rs

```diff
@@ -90,7 +93,16 @@ pub enum WcRequestMethods {
     CosmosGetAccounts,
     EthSignTransaction,
     EthSendTransaction,
-    PersonalSign,
+    EthPersonalSign,
+    // Notes (remove later):
+    // - This method will return the pubkey of each address :D
+    // - Wallets will return ALL addresses found in every purpose' derivation (44, 49, 84, 86), you need to filter for the ones the coin enabled with (or enable mixture of legacy and segwits?).
+    // - You want to listen to `bip122_addressesChanged` event (which has the same format as `getAccountAddresses` response)
+    //   but we can keep this a todo for later since we probably can manage without it for now.
+    // ref. https://docs.reown.com/advanced/multichain/rpc-reference/bitcoin-rpc
```

Comment:
Could you put the `TODO` keyword so it doesn't get lost?

```suggestion
    // TODO: remove this note later.
    // - This method will return the pubkey of each address :D
    // - Wallets will return ALL addresses found in every purpose' derivation (44, 49, 84, 86), you need to filter for the ones the coin enabled with (or enable mixture of legacy and segwits?).
    // - You want to listen to `bip122_addressesChanged` event (which has the same format as `getAccountAddresses` response)
    //   but we can keep this a todo for later since we probably can manage without it for now.
    // ref. https://docs.reown.com/advanced/multichain/rpc-reference/bitcoin-rpc
```

Reply:
i assumed i won't forget removing this since im around this area a lot anyway.
but yeah, better label it correctly because humans forget ;D
[028ff49](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2499/commits/028ff49f362c8d6d69ca43a4bdde0c329139d5a4)